airflow/providers/snowflake/hooks/snowflake_sql_api.py

```diff
@@ -137,7 +137,10 @@ def execute_query(
         conn_config = self._get_conn_params()
 
         req_id = uuid.uuid4()
-        url = f"https://{conn_config['account']}.{conn_config['region']}.snowflakecomputing.com/api/v2/statements"
+        url = (
+            f"https://{conn_config['account']}.{conn_config['region']}"
+            f".snowflakecomputing.com/api/v2/statements"
+        )
```

Comment:
Is this a new Black thing?

Reply:
No, I'm just not sure why it passed before.   It's 126 characters and should have raised an issue for being too long, no?  I can change it back since it isn't raising any errors.

Reply:
Could be a Black thing. I seem to remember it did not split long string literals at one point.